grpc/src/test/java/com/linecorp/armeria/common/grpc/GrpcMeterIdPrefixFunctionTest.java

```diff
@@ -48,7 +48,7 @@
 import com.linecorp.armeria.common.logging.RequestLogAccess;
 import com.linecorp.armeria.common.logging.RequestLogProperty;
 import com.linecorp.armeria.common.metric.MeterIdPrefix;
-import com.linecorp.armeria.common.metric.PrometheusMeterRegistries;
+import com.linecorp.armeria.common.prometheus.PrometheusMeterRegistries;
```

Comment:
Question) I'm wondering if to minimize breaking changes, we could keep the package structure the same. I think this could be done by also declaring a separate `prometheus0` module.

The upside would be:
- Guaranteed one-line breaking change for users since they only need to add a dependency. (no need to touch java code)
- The symmetry between `prometheus0` and `prometheus1` is just easier to reason about.
- Users using `prometheus1` don't need code for `prometheus0`

Reply:
> I think this could be done by also declaring a separate prometheus0 module.

Had a chat about this and I don't want to add `prometheus0` right now because we have to maintain the same classes in the core module and `prometheus0` module.
Having said this, we might introduce `prometheus0` later if we want to keep supporting Prometheus 0 dependency when we reach Armeria v2.0.

> we could keep the package structure the same.

We can't do this without breaking change for now. 😢